delta/kernel/src/main/scala/ch/epfl/bluebrain/nexus/delta/kernel/cache/KeyValueStore.scala

```diff
@@ -217,6 +217,14 @@ trait KeyValueStore[K, V] {
 
 object KeyValueStore {
 
+  def create[K, V](): KeyValueStore[K, V] = {
+    val cache: Cache[K, V] =
+      Caffeine
+        .newBuilder()
+        .build[K, V]()
+    new LocalCache(cache)
+  }
+
```

Comment:
I need another way to create a cache where it's build directly rather than deferred

Reply:
You should not need to introduce this method

Reply:
The other methods return an IO[KeyValueStore], how should I get just a KeyValueStore?

Reply:
Well, you just get a `IO[AuthTokenProvider]`, distage can handle that

Reply:
Yes but it's confusing that a constructor returns an IO when no IO is necessary. I propose that either we keep the new method, or we change the existing one to return normally rather than an IO

Reply:
Creating a cache is a side-effect, that is why it is wrapped in an IO

Reply:
Use the method which returns an IO